docs/testplan/SmartSwitch-PMON-testplan.md

```diff
@@ -0,0 +1,939 @@
+2024_05_22_19_08_52  reboot      Wed 22 May 2024 07:03:16 PM UTC  cisco   N/A
+
+On Switch: “show reboot-cause history SWITCH”
+
+Device    Name                 Cause       Time                             User    Comment
+--------  -------------------  ----------  -------------------------------  ------  ---------
+SWITCH    2024_05_31_00_33_30  reboot      Fri 31 May 2024 12:29:34 AM UTC  cisco   N/A
+SWITCH    2024_05_30_05_09_36  reboot      Thu May 30 05:04:00 UTC 2024     cisco   N/A
+SWITCH    2024_05_24_19_50_39  reboot      Fri 24 May 2024 07:45:40 PM UTC  cisco   N/A
+SWITCH    2024_05_24_04_13_41  Power Loss  N/A                              N/A     Unknown
+
+On Switch: “show reboot-cause history all"
+Device    Name                      Cause       Time    User    Comment
+--------  ------------------------  ----------  ------  ------  --------------------------------------------------------------------------
+SWITCH    2024_06_04_23_27_25       Power Loss  N/A     N/A     Unknown (First boot of SONiC version detached.12061-dirty-20240601.152750)
+DPU0      2024_06_04_23_27_28       Unknown     N/A     N/A     N/A
+DPU0      2024_05_29_21_28_22|DPU0  Unknown     N/A     N/A     N/A
+
+show reboot-cause history DPU0
+Device    Name                      Cause    Time    User    Comment
+--------  ------------------------  -------  ------  ------  ---------
+DPU0      2024_06_04_23_27_28       Unknown  N/A     N/A     N/A
+DPU0      2024_05_29_21_28_22       Unknown  N/A     N/A     N/A
+DPU0      2024_05_23_17_15_19       Unknown  N/A     N/A     N/A
+
+```
+#### Pass/Fail Criteria
+ *  A proper output should be delayed without any error as shown
+
+### 1.10 Check the show system-health summary CLI on the SWITCH
+
+#### Steps
+ * Issue this CLI “show system-health summary" on the switch
+ * Verify the health is reflected properly
+ * Test all, switch, DPU0 options by issuing “show system-health summary \<options\>”
+ * Verify the health is reflected properly for the given module
+ 
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch: “show system-health summary"
+
+System status summary
+
+  System status LED  green
+  Services:
+    Status: Not OK
+    Not Running: container_checker, lldp:lldpmgrd
+  Hardware:
+    Status: OK
+
+On Switch: “show system-health summary DPU0"
+
+DPU0
+System status summary
+
+  System status LED  green
+  Services:
+    Status: Not OK
+    Not Running: container_checker, macsec, dhcp_relay, mux, snmp, lldp, mgmt-framework, gnmi, swss:vlanmgrd, swss:vxlanmgrd, teamd:teammgrd, teamd:teamsyncd
+  Hardware:
+    Status: OK
+
+On Switch: “show system-health summary SWITCH"
+
+SWITCH
+System status summary
+
+  System status LED  green
+  Services:
+    Status: Not OK
+    Not Running: container_checker, lldp:lldpmgrd
+  Hardware:
+    Status: OK
+
+On Switch: “show system-health summary all"
+
+SWITCH
+System status summary
+
+  System status LED  green
```

Comment:
If the system status is "Not OK" the LED should be red I guess

Reply:
Fixed it.

Reply:
Fixed